lib/logger.js

```diff
@@ -6,18 +6,18 @@ var logger = new (winston.Logger)({});
 // Log into the console
 logger.add(winston.transports.Console, {
 	colorize: true,
-  level: (process.env.NODE_ENV === 'production') ? 'notice' : 'info'
+  level: (process.env.NODE_ENV === 'development') ? 'info' : 'warning'
```

Comment:
Luca I tweaked this as we don't have an "environment" concept for connectors. It'll now only log in development mode, which only happens on your local machine.